source/fundamentals/collations.txt

```diff
@@ -0,0 +1,531 @@
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start listIndexes
+   :end-before: end listIndexes
+
+The output of your code should contain the following:
+
+.. code-block:: none
+   :copyable: false
+
+   { ...
+     "collation": { "locale": "en_US", ... }
+     ...
+   }
+
+.. note::
+
+   You can assign a default collation to a collection only when you create
+   that collection. To use collations for supported operations on an
+   existing collection, create an index with the collation and specify the
+   same collation in your operations on it.
+
+.. _collations-index:
+
+Index
+~~~~~
+
+You can specify a collation when you create a new index on a collection.
+The index stores an ordered representation of the documents in the
+collection so your operation does not need to perform the ordering
+in-memory. To use the index, your operation must meet the following
+criteria:
+
+- The operation uses the same collation as the one specified in the index.
+- The operation is covered by the index containing the collation.
+
+The following code snippet shows how you can create an index on the "name"
+field with the "en_US" locale collation in ascending order:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start createIndex
+   :end-before: end createIndex
+
+To check whether you created the collation successfully, retrieve a list
+of the indexes on that collection as follows:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start listIndexes
+   :end-before: end listIndexes
+
+The output of the code above should contain the following:
+
+.. code-block:: none
+   :copyable: false
+
+   { ...
+     "collation": { "locale": "en_US", ... }
+     ...
+   }
+
+The following code snippet shows an example operation that specifies the
+same collation and is covered by the index we created above:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start indexOperation
+   :end-before: end indexOperation
+
+.. _collations-operation:
+
+Operation
+~~~~~~~~~
+
+You can override the default collation on a collection by passing the
+new collation as a parameter to one of the supported operations. However,
+since the operation does not use an index, the operation may not perform
+as well as one that is covered by an index. For more information on the
+disadvantages of sorting operations not covered by an index, see the server
+manual page on :manual:`Use Indexes to Sort Query Results </tutorial/sort-results-with-indexes/>`.
+
+The following code snippet shows an example query operation that specifies
+the Icelandic ("is") collation that is not covered by the default collation
```

Comment:
I agree the length can make the sentence more complex. Since the sentence is describing a single item, I think dividing it makes the sentences too short which can be misinterpreted as being talked down to. E.g. "The brown dog that lives with its owner ran to the store." vs "The brown dog lives with its owner. The brown dog ran to the store." Instead, I'll present multiple (including information that wasn't presented explicitly before) conditions which can be listed in fragments.

Reply:
I like the edits! Solves the issue I experienced and avoids a potentially patronizing tone. Resolving conversation.